assets/src/edit-story/app/rightClickMenu/provider.js

```diff
@@ -251,6 +256,31 @@ function RightClickMenuProvider({ children }) {
     });
   }, [selectedElement, selectedElementAnimations]);
 
+  const selectedElementId = selectedElement?.id;
+  const pushUpdate = useCallback(
+    (update, commitValues) => {
+      if (selectedElementId) {
+        updateElementsById({
+          elementIds: [selectedElementId],
+          properties: (element) => {
+            const updates = updateProperties(element, update, commitValues);
+            const sizeUpdates = getUpdatedSizeAndPosition({
+              ...element,
+              ...updates,
+            });
+            return {
+              ...updates,
+              ...sizeUpdates,
+            };
+          },
+        });
+      }
+    },
+    [selectedElementId, updateElementsById]
+  );
```

Comment:
🤔 do you think we should be using `pushUpdate` for other `updateElementsById`s in this file? I'm a little confused about what the difference is

Reply:
Is it just teh size and position updates?

Reply:
Yea I think we're fine with leaving it how it is for other element updates in this file. 

The only reason I have it here is I needed to alter the fn signature around `updateElementsById` to fit into all the preset callbacks. No need to wrap the other updates to elements with extra layers if not needed imo. 

Also, yea, the `getUpdatedSizeAndPosition` is specific to text updates, so no need to use it anywhere else.